definitions/tools/mutect.cwl

```diff
@@ -21,15 +21,8 @@ requirements:
             
             TUMOR=`(samtools view -H $3 | grep "^@RG" | sed "s/.*SM:\([^\t]*\).*/\1/g" | head -n 1)` #Extracting the sample name from the TUMOR bam.
             NORMAL=`(samtools view -H $4 | grep "^@RG" | sed "s/.*SM:\([^\t]*\).*/\1/g" | head -n 1)` #Extracting the sample name from the NORMAL bam.
```

Comment:
Should this regex be changed to match the one in the filter_docm tool that you used, for consistency?

Reply:
Agreed, fixed in 29a3f2c